### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by anitaposliedova**

### CODE REVIEW

**Review by Oleksandr563487953 — APPROVED** [review]

File: tests/test_main.py [hunk 1]
```diff
@@ -1 +1,39 @@
...
+        ),
+        pytest.param(
+            1.30, 1.26,
+            "Do nothing",
+            id="test without changes -5%"
+        ),
+        pytest.param(
+            1.8, 1.84,
+            "Do nothing",
+            id="test without changes +5%)"
```
Comment (MasterpieceElbow) [inline tests/test_main.py:27]: Add boundary conditions, you don't check when `get_exchange_rate_prediction` return value equals exactly 1.05 or 0.95 * value passed in `cryptocurrency_action`.

**Review by MasterpieceElbow — CHANGES_REQUESTED** [review]

File: tests/test_main.py (hunk 1 above)
Comment (MasterpieceElbow) [reply tests/test_main.py:27]: You have not fixed my comment. 1.8 / 1.85 is 0.97297... not **exactly** 0.95.

**Review by MasterpieceElbow — CHANGES_REQUESTED** [review]

**Review by MasterpieceElbow — CHANGES_REQUESTED** [review]
